### DISCUSSION TITLE
关于 qlog 的 qlog_format 设置

### DISCUSSION THREAD

**Discussion by nixwl** [Tencent/tquic#150] [Q&A (问题和回答)] [answered]
我希望使用 qvis 对 进行可视化分析，但是 qvis 支持的 qlog 需要满足 `{"qlog_format": "JSON",`， TQUIC 的源码中 `src\qlog` 使用的是JSON-SEQ， 我如何将其改为 JSON。或者说还有什么方法直接得到 qlog_format 为 JSON 的文件

**Comment by iyangsj** [comment, marked as answer]
TQUIC下个版本v0.7.0会提供兼容转换工具和说明文档

**Reply by nixwl** [reply]
好的，非常感谢

**Comment by nixwl** [comment]
关于 JSON-SEQ 转 JSON 的问题官方已经解决了：使用`[ /tools/script/tquic_qvis.sh]`即可 ，[链接如下：](https://github.com/Tencent/tquic/blob/develop/tools/script/tquic_qvis.sh)

转换完成后，就能直接导入 qvis 使用了：website：[https://qvis.quictools.info/#/files](https://qvis.quictools.info/#/files)

然后提供一下我的示例：
![image](https://github.com/Tencent/tquic/assets/80773211/f5531c1d-2d01-4714-b67b-6a57d52302c4)

> 关于 qvis 导入 JSON-SEQ [存在问题这个事，我已经向qvis社区反应了，大家可以参照：[#82](https://github.com/quiclog/qvis/issues/82)
